lavalink/player_manager.py

```diff
@@ -143,8 +162,8 @@ async def move_to(self, channel: discord.VoiceChannel, deafen: bool = False):
         ----------
         channel : discord.VoiceChannel
         """
-        if channel.guild != self.channel.guild:
-            raise TypeError("Cannot move to a different guild.")
+        if channel.guild != self.guild:
+            raise TypeError(f"Cannot move {self} to a different guild.")
```

Comment:
Optional, shows the intent more clearly.
```suggestion
            raise TypeError(f"Cannot move {self!r} to a different guild.")
```